src/govuk/components/character-count/character-count.mjs

```diff
@@ -26,7 +27,21 @@ function CharacterCount ($module, config) {
   }
 
   var defaultConfig = {
-    threshold: 0
+    threshold: 0,
+    i18n: {
+      // Characters
+      charactersUnderLimitZero: 'You have %{count} characters remaining',
```

Comment:
Thought of renaming that one `charactersAtLimit` rather than piggyback on the `charactersUnderLimit`. This would have had impact on #2701 if it got started in parallel so stuck with the naming from the spike. Happy to add an extra commit to separate things.

Reply:
`charactersAtLimit` makes sense to me 👍🏻 

Do we need to change anything about the implementation as that is effectively a separate translation that isn't pluralised?

Reply:
Yup, it requires a little update to `formatCountMessage` in the lines of:

```js
if (remainingNumber === 0) {
    var translationKey = type + 'AtLimit'
	return I18n.t(translationKey)
}

// Do things as it does now otherwise
```

So nothing too wild 😄

Reply:
@claireashworth Because we're having a special option for when 0 words or characters are remaining now, we are no longer tied to the "You have <number> words/characters remaining" sentence structure when users hit the limit. 

Do you think it's worth changing the wording for that case, for ex to using only words "You have no words/characters remaining"? Thought behind that, besides it helping me with testing (😆), it that it'd also make that special situation stand out from when you're not quite there/past the limit because there'd be no longer a number in the sentence.

Reply:
'You have 0 words/characters remaining' keeps the sentence structure consistent.
I don't think 'You have no words/characters remaining' makes it stand out more. A 'more correct' way of phrasing it would be 'You do not have any words/characters remaining', but that is quite long to put under a text box.
Maybe swapping to 'zero' instead of '0' would work as it could not be mistaken for 'o' and is more characters which makes it more visible?

Reply:
Fair point on the 'standing out' not being massive with the 'no', that was a bit of a tenuous argument on my part 😬 . 'zero' is a much better option for that indeed, thanks for the idea 😄

Do we have any guidance on whether we'd prefer consistency (I can always turn my tests a different way to accomodate the sentences being the same) over disambiguating the '0'/switching to fully written number in that case (which helps my testing, but I don't want it to be at the detriment of users)?

Reply:
Nothing specific in guidance about consistency. It's generally better to be consistent with routine things and seeing a '0' would probably be accepted by the users as logical (I obviously can't say that with certainty as I haven't researched it). If using 'zero' for testing purposes helps, then in my opinion it shouldn't confuse users.

The only other point to note is that the government style guide specifically says "Use a 0 where there’s no digit before the decimal point." So if we stick to the government's chosen style we have to use '0'. If we use 'zero' we should be able to demonstrate that it's a better choice in these circumstances.

Reply:
I went and reverted to `0` in number in the end, keeping thing consistent with not only the rest of the messages but the current behaviour as well. Thanks for all the clarifications 😄